Jenkinsfile

```diff
@@ -55,13 +55,13 @@ try {
   
     // The empty '' results in using the default registry: https://index.docker.io/v1/
     docker.withRegistry('', 'docker') {
-      def oppossumCI = docker.image('hyrise/opossum-ci:20.04');
-      oppossumCI.pull()
+      def hyriseCI = docker.image('hyrise/hyrise-ci:20.04');
+      hyriseCI.pull()
 
       // LSAN (executed as part of ASAN) requires elevated privileges. Therefore, we had to add --cap-add SYS_PTRACE.
       // Even if the CI run sometimes succeeds without SYS_PTRACE, you should not remove it until you know what you are doing.
       // See also: https://github.com/google/sanitizers/issues/764
-      oppossumCI.inside("--cap-add SYS_PTRACE -u 0:0") {
+      hyriseCI.inside("--cap-add SYS_PTRACE -u 0:0 --tty") {
```

Comment:
Why the addition?

Reply:
I added it in an attempt to solve the CI error which was then solved by Marcel.
Thanks for the catch!